docs-site/docs/extend/graphql.md

```diff
@@ -0,0 +1,215 @@
+
+## Accessing GraphIQL
+
+The easiest way to use the GraphQL API is through `GraphIQL` using the following steps.
+
+1. Sign into IRIDA Next `http://my.irida.server/users/sign_in`
+2. Go to the GraphIQL url `http://my.irida.server/graphiql`
+
+You will be greeted with the in browser tool for writing, validating and testing GraphQL queries.
+
+For more on GraphIQL see [the official docs](https://graphql-dotnet.github.io/docs/getting-started/graphiql/)
+
+## GraphQL Examples
+
+<table>
+<tr>
+<td> Example </td> <td> Query </td> <td> Response </td>
+</tr>
+<tr>
+<td> Get Current User </td>
+<td>
+
+```graphql
+query {
+  currentUser {
+    email
+  }
+}
+```
+
+</td>
+<td>
+
+```json
+{
+  "data": {
+    "currentUser": {
+      "email": "admin@email.com"
+    }
+  }
+}
+```
+
+</td>
+</tr>
+<tr>
+<td> Get Samples </td>
+<td>
+
+```graphql
+query {
+  samples (first:5){
+    nodes{
+      name
+      id
+    }
+  }
+}
+```
+
+</td>
+<td>
+
+```json
+{
+  "data": {
+    "samples": {
+      "nodes": [
+        {
+          "name": "Bacillus anthracis/Outbreak 2022 Sample 1",
+          "id": "gid://irida/Sample/1"
+        },
+        {
+          "name": "Bacillus anthracis/Outbreak 2022 Sample 2",
+          "id": "gid://irida/Sample/2"
+        },
+        {
+          "name": "Bacillus anthracis/Outbreak 2022 Sample 3",
+          "id": "gid://irida/Sample/3"
+        },
+        {
+          "name": "Bacillus anthracis/Outbreak 2022 Sample 4",
+          "id": "gid://irida/Sample/4"
+        },
+        {
+          "name": "Bacillus anthracis/Outbreak 2022 Sample 5",
+          "id": "gid://irida/Sample/5"
+        }
+      ]
+    }
+  }
+}
+```
+
+</td>
+</tr>
+
+<tr>
+<td> Get a specific samples details </td>
+<td>
+
+```graphql
+query {
+  node(id: "gid://irida/Sample/1"){
+    ... on Sample{
+      id,
+      name
+      description
+      metadata
+      project{
+        id,
+        fullPath
+      }
+    }
+  }
+}
+```
+
+</td>
+<td>
+
+```json
+{
+  "data": {
+    "node": {
+      "id": "gid://irida/Sample/1",
+      "name": "Bacillus anthracis/Outbreak 2022 Sample 1",
+      "description": "This is a description for sample Bacillus anthracis/Outbreak 2022 Sample 1.",
+      "project": {
+        "id": "gid://irida/Project/1",
+        "fullPath": "bacillus/bacillus-anthracis/outbreak-2022"
+      },
+      "metadata": {
+        "age": 40,
+        "food": "Cheeseburger",
+        "onset": "2022-06-21",
+        "WGS_id": 6862301436,
+        "gender": "Female",
+        "country": "Gabon",
+        "patient_age": 8,
+        "patient_sex": "Male",
+        "earliest_date": "2022-10-03",
+        "NCBI_ACCESSION": "NM_7807606.5"
+      }
+    }
+  }
+}
+```
+
+</td>
+</tr>
+</table>
+
+## Authentication with Personal Access Tokens
+
+To use the GraphQL API with [any of the GraphQL Clients](https://graphql.org/code/#graphql-clients), you will need to generate a Personal Access Token.
+
+### Navigate to the Access Tokens screen
```

Comment:
**nitpick (blocking):** Please remove the screenshots and just use descriptive text instead, like https://phac-nml.github.io/irida-next/docs/user/organization/members/group-and-project-members#add-members-to-a-project

Reply:
Changed.